Kitodo/src/main/java/de/sub/goobi/beans/Benutzer.java

```diff
@@ -176,12 +175,18 @@ public void setPasswort(String inpasswort) {
 		this.passwort = inpasswort;
 	}
 
+	/**
+	 * @return add description
```

Comment:
![add](https://cloud.githubusercontent.com/assets/9614459/21314758/0b07dcf0-c5f9-11e6-8d26-78f1dfdac1af.png)
 After doing it we would get about 2 000 (maybe even more) TODOs.